tests/Integration/Cache/RedisCacheLockTest.php

```diff
@@ -72,6 +72,24 @@ public function test_concurrent_redis_locks_are_released_safely()
         $this->assertFalse(Cache::store('redis')->lock('foo')->get());
     }
 
+    public function test_redis_locks_with_failed_block_callback_are_released()
+    {
+        Cache::store('redis')->lock('foo')->forceRelease();
+
+        $firstLock = Cache::store('redis')->lock('foo', 10);
+        try {
```

Comment:
Add a white line above the try block.

```suggestion

        try {
```

Reply:
Done